packages/utils/src/lib/reports/utils.unit.test.ts

```diff
@@ -311,11 +311,9 @@ describe('countWeightedRefs', () => {
 describe('compareIssueSeverity', () => {
   it('should order severities in logically ascending order when used as compareFn with .sort()', () => {
     const severityArr = ['error', 'info', 'warning'] satisfies IssueSeverity[];
```

Comment:
Not really related, but still :grin: 

```suggestion
    const severityArr: IssueSeverity[] = ['error', 'info', 'warning'];
```

Reply:
could that be a linting rule?

Reply:
It probably could be but did not find any existing rule.

Reply:
Responded here: https://github.com/code-pushup/cli/pull/548#issuecomment-1985379373